src/openforms/js/components/admin/submissions/filter.js

```diff
@@ -0,0 +1,47 @@
+const init = async () => {
+  // Retrieve the filter link that displays only triggered rules
+  const triggeredLink = document.querySelector('.logic-rules-filter > .selected > a');
+
+  // Only triggered rules are displayed
+  document
+    .querySelectorAll('.logic-logs__col-body--trigger-' + triggeredLink.dataset.trigger)
+    .forEach(trigger => {
+      trigger.parentNode.classList.remove('logic-logs__row-body--hidden');
+    });
+};
+
+document.querySelectorAll('.logic-rules-filter__link').forEach(link => {
```

Comment:
you typically only want to start performing DOM queries when the DOM has finished loading, so you'd move this entire block inside the `init` function